src/syntax_highlight/null.rs

```diff
@@ -0,0 +1,170 @@
+use liquid;
+use liquid::Token::{self, Identifier};
+use liquid::lexer::Element::{self, Expression, Tag, Raw};
+
+use pulldown_cmark as cmark;
+
+use error;
+
+pub fn has_syntax_theme(_name: &str) -> error::Result<bool> {
+    bail!("Themes are unsupported in this build.");
+}
+
+pub fn list_syntax_themes<'a>() -> Vec<&'a String> {
+    vec![]
+}
+
+pub fn list_syntaxes<'a>() -> Vec<String> {
+    vec![]
+}
+
```

Comment:
I think we should bail on `list_syntax_themes` and `list_syntaxes`, like with `has_syntax_theme`.  These two functions are intended to only drive the CLI, and shouldn't be used anywhere else in cobalt.  But that can be the focus of another patch.